java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -10,81 +10,81 @@
 import java.util.UUID;
 
 public class TransactionTestUtilities {
-    private static final String key1 = "{key}" + UUID.randomUUID();
-    private static final String key2 = "{key}" + UUID.randomUUID();
-    private static final String key3 = "{key}" + UUID.randomUUID();
-    private static final String key4 = "{key}" + UUID.randomUUID();
-    private static final String key5 = "{key}" + UUID.randomUUID();
-    private static final String key6 = "{key}" + UUID.randomUUID();
-    private static final String key7 = "{key}" + UUID.randomUUID();
-    private static final String key8 = "{key}" + UUID.randomUUID();
-    private static final String value1 = UUID.randomUUID().toString();
-    private static final String value2 = UUID.randomUUID().toString();
-    private static final String value3 = UUID.randomUUID().toString();
-    private static final String field1 = UUID.randomUUID().toString();
-    private static final String field2 = UUID.randomUUID().toString();
-    private static final String field3 = UUID.randomUUID().toString();
+    private static final String stringKey1 = "stringKey1-" + UUID.randomUUID();
```

Comment:
As discussed during our meeting, this can cause keys to span multiple slots and nodes.  A single transaction must be executed by a single node, so we may have problems if you rename the keys.  
i'd suggest reverting the name change.

Reply:
I found a way how to bypass that